string out_dir
uint16 data_point
uint16 frame_count
float64[] target_config
---
uint16 captured_frames
